intro_to_r/intro_to_r.Rmd

```diff
@@ -157,7 +154,7 @@ install.packages("mosaic")
 Note that you will only have to do this *once*. However, once the package is installed, you will have to load it into the current workspace before it can be used. 
 
 ```{r require-mosaic, message=FALSE}
-require(mosaic)
+library(mosaic)
```

Comment:
Since [this](http://stackoverflow.com/questions/5595512/what-is-the-difference-between-require-and-library) suggests no compelling reason to use `require`, we should be consistent with the rest of the lab.